client/components/Events/EventDateTimeColumn.tsx

```diff
@@ -23,7 +24,7 @@ export class EventDateTimeColumn extends React.PureComponent<IProps> {
             return (
                 <Column border={false} className="flex-justify--start sd-padding-t--1">
                     <Row classes="sd-margin--0">
-                        <EventDateTime item={this.props.item} />
+                        <EventDateTime item={this.props.item} planningItem={this.props.planningItem} />
```

Comment:
Why is this component called `EventDateTime` and we're passing an event item and a planningItem? If it's meant to show both planning & event item times you should rename it accordingly